subsys/bluetooth/mesh/Kconfig

```diff
@@ -129,16 +129,6 @@ choice TFM_PROFILE_TYPE
 	default TFM_PROFILE_TYPE_NOT_SET
 endchoice
 
-config BT_MESH_DFU_METADATA_ON_BUILD
```

Comment:
We want this Kconfig option. Please revert this change and the print in `mesh_dfu_metadata.py`. We will fix it separately.

Reply:
This option has done nothing for half a year

Reply:
That is a bug. This doesn't mean it should be removed.

Reply:
No it's not a bug, it was transitioned to sysbuild https://github.com/nrfconnect/sdk-nrf/blob/main/sysbuild/Kconfig.zip#L42